.github/workflows/on-qa-firebase-distribution.yml

```diff
@@ -0,0 +1,64 @@
+name: Build QA Release and Distribute on Firebase
+on:
+    workflow_dispatch:
+        inputs:
+            buildType:
```

Comment:
Am not sure that this would work.  I would try something like 
`    inputs:
        environment:
          type: choice
          description: 'Select environment'
          required: true
          options:
            - Playground
            - Dev
            - Production
`  
Also, if needed,  we could add another option for the Release or Debug flavor.

Reply:
There is a specific `environment` type, not sure how it works though I've read it on documentation:
https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/

Reply:
Would not have any problem but feel free to change it in the previous value.